server/opts.go

```diff
@@ -157,6 +157,7 @@ type Options struct {
 	NoLog                 bool          `json:"-"`
 	NoSigs                bool          `json:"-"`
 	NoSublistCache        bool          `json:"-"`
+	NoHeaderSupport       bool          `json:"-"`
```

Comment:
Do you envision "new" servers not supporting header, or is it more a way to test mix of old and new? If so, we could make it a "private" (non exported) option.

Reply:
Mostly testing TBH/